app.rb

```diff
@@ -0,0 +1,123 @@
+require './book'
+require './classroom'
+require './rental'
+require './person'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    @class = Classroom.new('class 8')
+  end
+
+  def get_user_option(option)
+    case option
+    when '1'
+      all_books
+    when '2'
+      all_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      rentals_by_id
+    else
+      puts 'Wrong option'
+      puts '--------------------------------------'
+      puts 'Please enter a number between 1 and 7'
+    end
+  end
```

Comment:
- [ ] Following the `Single responsibility rule`, all requests for input and input validation should placed in classes/methods whose responsibilities are to request or validate input. Please checkout this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) and kindly fix that.